spark/dl/src/test/scala/com/intel/analytics/bigdl/keras/nn/LocallyConnected2DSpec.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.keras.nn
+
+import com.intel.analytics.bigdl.keras.KerasBaseSpec
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, DataFormat}
+import com.intel.analytics.bigdl.nn.keras.{LocallyConnected2D, Sequential => KSequential}
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.utils.Shape
+
+class LocallyConnected2DSpec extends KerasBaseSpec {
+
+  def weightConverter(data: Array[Tensor[Float]]): Array[Tensor[Float]] = {
+    val out = new Array[Tensor[Float]](data.length)
+    val d1l: Int = data(0).size(1)
+    val d2l: Int = data(0).size(2)
+    val d3l: Int = data(0).size(3)
+    out(0) = Tensor(d1l, d3l, d2l)
+    val page: Int = d2l * d3l
+    for (i <- 0 to d1l * d2l * d3l - 1) {
+      val d1 = i / page + 1
+      val d2 = (i % page) / (d3l) + 1
+      val d3 = (i % page) % d3l + 1
+      val v = data(0).valueAt(d1, d2, d3)
+      out(0).setValue(d1, d3, d2, v)
+    }
+    if (data.length > 1) {
+      out(1) = data(1)
+    }
+    out
+  }
+
+  "LocallyConnected2D" should "be the same as Keras" in {
+    val kerasCode =
+      """
+        |input_tensor = Input(shape=[3, 4, 5])
+        |input = np.random.random([2, 3, 4, 5])
+        |output_tensor = LocallyConnected2D(64, 3, 3, dim_ordering="th", activation="relu")(input_tensor)
+        |model = Model(input=input_tensor, output=output_tensor)
+      """.stripMargin
+    val seq = KSequential[Float]()
+    val layer = LocallyConnected2D[Float](
+      nbFilter = 64,
+      nbRow = 3,
```

Comment:
Make this cleaner, like `LocallyConnected2D(64, 3, 3, dim_ordering="th", activation="relu")`, define it in one line.